r.in.wcs.worker/r.in.wcs.worker.py

```diff
@@ -161,10 +173,58 @@ def main():
     os.remove(tif)
     tif = tif.replace(".0", ".tif")
     RM_FILES.append(tif)
-    try:
-        urlretrieve(url, tif)
-    except URLError:
-        grass.fatal(_(f"Failed to reach the server.\nURL: {url}"))
+
+    num_retry_no_connection = 0
+    while num_retry_no_connection <= num_retry_max:
+        try:
+            num_retry_unstable_connection = 0
+            urlretrieve(url, tif)
```

Comment:
```suggestion
            urlretrieve(url, tif)
```
Do not reset `num_retry_unstable_connection` to zero every time this block is entered.